js/process_test.ts

```diff
@@ -205,6 +220,57 @@ testPerm({ run: true }, async function runStderrOutput(): Promise<void> {
   p.close();
 });
 
+testPerm(
+  { run: true, write: true, read: true },
+  async function runRedirectStdoutStderr(): Promise<void> {
+    const tempDir = await makeTempDir();
+    const fileName = tempDir + "/redirected_stdio.txt";
+    const file = await open(fileName, "w");
+
+    const p = run({
+      args: [
+        "python",
+        "-c",
+        "import sys; sys.stderr.write('error\\n'); sys.stdout.write('output\\n');"
+      ],
+      stdout: file,
+      stderr: file.rid
```

Comment:
Are you testing that you can use both an rid and a File object as a parameter here? If so please state that. It's confusing. I'm not sure I agree that the options object should have the ability to take both an rid and a File object.

Reply:
Removed `File` as option for `stdin/stdout/stderr`